agent/consul/server.go

```diff
@@ -114,8 +115,7 @@ type Server struct {
 	// acls is used to resolve tokens to effective policies
 	acls *ACLResolver
 
-	aclAuthMethodValidators    map[string]*authMethodValidatorEntry
-	aclAuthMethodValidatorLock sync.RWMutex
+	aclAuthMethodValidators authmethod.AuthMethodCache
```

Comment:
trivial: maybe rename `authmethod.AuthMethodCache` to `authmethod.Cache` to remove the identifier stutter

Reply:
will do